widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.kt

```diff
@@ -1177,4 +1179,18 @@ class ChatView(context: Context, attrs: AttributeSet?, defStyleAttr: Int, defSty
         unreadIndicatorTheme.background?.primaryColor?.also(binding.newMessagesIndicatorCard::setCardBackgroundColor)
     }
 
+    private fun showChatUnavailableView() {
+        alertDialog = Dialogs.showMessageCenterUnavailableDialog(context, theme) {
+            hideChatControls(it.window?.decorView?.height ?: 0)
+        }
+    }
+
+    private fun hideChatControls(dialogHeight: Int) {
+        binding.apply {
+            chatRecyclerView.updatePadding(bottom = dialogHeight)
+            chatRecyclerView.scrollBy(0, dialogHeight)
+            groupChatControls.isGone = true
+        }
+    }
+
 }
```

Comment:
Please look over the generated filed for missing newline at the end of the files